### ISSUE TITLE
Error: [HPM] Invalid context. Expecting something like: "/api" or ["/api", "/ajax"]

### ISSUE THREAD

**Issue by MehrCurry** [dlizarra/spring-boot-react-webpack-starter#1]
I am doing my first steps to create a react app on top of spring boot and found this project which seems to be a good starting point for me.

But after i get everything up an running i get an error message when calling http://localhost:9090

I seems to be something to do with `http-proxy-middleware` but i have no idea how to fix it.

Any hints for me?

```
Error: [HPM] Invalid context. Expecting something like: "/api" or ["/api", "/ajax"]
    at Object.matchContext [as match] (/Users/guido/wrk/starter/src/main/frontend/node_modules/http-proxy-middleware/lib/context-matcher.js:40:11)
    at shouldProxy (/Users/guido/wrk/starter/src/main/frontend/node_modules/http-proxy-middleware/lib/index.js:70:31)
    at middleware (/Users/guido/wrk/starter/src/main/frontend/node_modules/http-proxy-middleware/lib/index.js:38:13)
    at Layer.handle [as handle_request] (/Users/guido/wrk/starter/src/main/frontend/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/Users/guido/wrk/starter/src/main/frontend/node_modules/express/lib/router/index.js:312:13)
    at /Users/guido/wrk/starter/src/main/frontend/node_modules/express/lib/router/index.js:280:7
    at Function.process_params (/Users/guido/wrk/starter/src/main/frontend/node_modules/express/lib/router/index.js:330:12)
    at next (/Users/guido/wrk/starter/src/main/frontend/node_modules/express/lib/router/index.js:271:10)
    at /Users/guido/wrk/starter/src/main/frontend/node_modules/webpack-dev-server/lib/Server.js:183:7
    at Layer.handle [as handle_request] (/Users/guido/wrk/starter/src/main/frontend/node_modules/express/lib/router/layer.js:95:5)
```

**Comment by dlizarra** [comment]
I'll try to reproduce the error when I get home, but this must come from the webpack devServer configuration. [Here](https://github.com/dlizarra/spring-boot-react-webpack-starter/blob/master/src/main/frontend/webpack.config.js#L38) we can specify which paths we want to be proxied to our Spring Boot backend server. Can you try changing `'/*'` to `/` or to `**` and running it again? I believe this will fix the error.

**Comment by dlizarra** [comment]
I updated the project with the fix: d4a030b

`/*` was not valid syntax. For proxying all requests we can use for example `/` or the wildcard `**` as stated in  `http-proxy-middleware` [docs](https://github.com/chimurai/http-proxy-middleware#context-matching).

Thanks for noticing!

**Comment by MehrCurry** [comment]
Thanks, the fix works for me
